packages/gitlab-backend/README.md

```diff
@@ -0,0 +1,14 @@
+# gitlab-annotation-filler
```

Comment:
We should do the README before publishing

Reply:
Yes, sure, I have to refactor all the READMEs. Because of this feature, we are also changing the configuration.